tasks/task035_winogrande_question_modification_person.json

```diff
@@ -55,7 +55,7 @@
         },
         {
             "input": "Context word: step. \nQuestion: PersonX was always ahead of PersonY, as _ walked with a quick step. \nAnswer: PersonX.",
-            "output": "PersonY was always ahead of PersonY, as _ walked with a quick step .",
+            "output": "PersonY was always ahead of PersonX, as _ walked with a quick step .",
```

Comment:
@swarooprm since you were skeptic on this, here is an example of several nice suggestions by a crowdworkers: 

> Def: The formatting was hard to read; numbered lists typically have line breaks between list items.There were also grammatical errors ("should be an well-agreed answer", "atleast").The clarity could have been improved in some areas. For instance, "void repeating the same style or phrase in generating your modified question e.g. this task can be always solved using a simple negation i.e. by adding not, never, etc. Instead, try to increase the word diversity." is a confusing sentence, particularly because of the use of both "i.e." and "e.g."Also, the using real names examples are kind of unnecessary ("Donald Trump or Putin").
> P examples: The grammar is weird at times ("replacing the trigger word 'sympathetic' by 'stern' "). I would think you might use the word "with" instead of "by."
> N examples: The first example didn't really make sense because of (presumably) a typo: "PersonY was always ahead of PersonY, as _ walked with a quick step ."